controller/reviewController.js

```diff
@@ -23,8 +24,36 @@ import authenticate from '../middlewares/authMiddlewares.js';
   }
 };
 
+
+/**
+ * Retrieves the reviews of the place with the given placeId.
+ */
+const getAllReviewsByPlace = async (req, res) => {
+  try {
+    //   authenticate(req, res, next);
+    const response = await getAllReviewsByPlaceHandler(req.params.placeId);
+    res.status(response.status || StatusCodes.OK).json(response);
+  } catch (err) {
+    res.status(StatusCodes.UNAUTHORIZED).json({ message: err.message || 'User not authenticated' });
+  }
+}
+
+/**
+ * Retrieves the reviews of the user with the given username.
+ */
+const getAllReviewsByUser = async (req, res) => {
+  try {
+    //   authenticate(req, res, next);
+    const response = await getAllReviewsByUserHandler(req.params.username);
+    res.status(response.status || StatusCodes.OK).json(response);
+  } catch (err) {
+    res.status(StatusCodes.UNAUTHORIZED).json({ message: err.message || 'User not authenticated' });
+  }
+}
+
 const reviewController = app => {
-  // app.get('/places/details/:placeId/reviews', getReviews)
+  app.get('/places/details/:placeId/reviews', getAllReviewsByPlace)
```

Comment:
This endpoint IMO should belong to reviews. So URL should look something like '/reviews/place/:placeId'. Thoughts?
Same for getting reviews by username as well. '/reviews/user/:username'.
For the sake of consistency, we can also refactor the POST endpoint to '/reviews'